### PR TITLE
add cards to resources page and modal on clicking on them

### PR DESCRIPTION
**Pull request by shubhshrma**
Fixes issue #52 

Changes: 

1. Add resources.css to ```staticFiles/home``` and change resources.html

2. add a css_include block in base.html

3. you have to include jQuery before bootstrap  js in base.html

Screenshots for the change:

image

image

### CODE REVIEW

File: staticFiles/home/resources.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    overflow-x:hidden;
+}
+.card {
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    max-width: 45%;
+    max-height: 300px;
+    border-radius: 5px;
+    margin: 20px auto;
+
```
Comment (divyam3897) [inline staticFiles/home/resources.css]: Remove empty line.

File: staticFiles/home/resources.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+body{
+    overflow-x:hidden;
```
Comment (divyam3897) [inline staticFiles/home/resources.css]: Add space after ":"

File: staticFiles/home/resources.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+img {
+    border-radius: 5px 5px 0 0;
+    width: 100%;
+    max-height: 100%;
+    
+}
+
+.container {
+    padding: 2px 16px;
+
```
Comment (divyam3897) [inline staticFiles/home/resources.css]: Remove empty lines.

File: staticFiles/home/resources.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+.card:hover {
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    }
+
+img {
+    border-radius: 5px 5px 0 0;
+    width: 100%;
+    max-height: 100%;
+
```
Comment (divyam3897) [inline staticFiles/home/resources.css]: Remove empty line.

File: templates/base.html [hunk 5]
```diff
@@ -8,7 +8,7 @@
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
     <title>{% block title %}Programming Club{% endblock %}</title>
     <!-- Bootstrap -->
-    <link rel="stylesheet" type="text/css" href="{% static 'home/base.css' %}"/>
+    {% block css_include %}<link rel="stylesheet" type="text/css" href="{% static 'home/base.css' %}"/>{% endblock %}
```
Comment (divyam3897) [inline templates/base.html]: why this change here?

File: templates/base.html [hunk 6]
```diff
@@ -18,15 +18,16 @@
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
 </head>
-    <!-- Latest compiled and minified JavaScript -->
```
Comment (divyam3897) [inline templates/base.html]: why this change?

File: templates/home/resources/resources.html [hunk 7]
```diff
@@ -4,3 +4,105 @@
 {% block nav_resources %}
     class="active"
 {% endblock %}
+{% block css_include %}
+<link rel="stylesheet" type="text/css" href="{% static 'home/resources.css' %}"/>
+{% endblock %}
+{% block content %}
+<div class="row">
+<div class="col-md-4 col-sm-12 col-xs-12">
+<div class="card" data-toggle="modal" data-target="#myModalCpp">
```
Comment (divyam3897) [inline templates/home/resources/resources.html]: Instead of redundant code, use a for loop see https://stackoverflow.com/questions/10326159/django-for-loop-in-a-html-template-page-newbie

**Comment by sagar-kalra** [comment]
@shubhshrma  still working on it?

**Comment by shubhshrma** [comment]
@sagar-kalra closing this.Sorry for the inconvenience,been busy.
